src/shared/components/Header/PayHeader/PayHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import * as styles from './PayHeader.css';
+import BackIcon from '@/shared/assets/icon/ic_back_white_24.svg';
+import CloseIcon from '@/shared/assets/icon/ic_close_white_24.svg';
+import Step1Icon from '@/shared/assets/icon/ic_header_1_step.svg';
+import Step2Icon from '@/shared/assets/icon/ic_header_2_step.svg';
+import Step3Icon from '@/shared/assets/icon/ic_header_3_step.svg';
+interface Props extends React.HTMLProps<HTMLDivElement> {
+  step: 1 | 2 | 3;
+  onBackClick?: () => void;
+  onCloseClick?: () => void;
+}
+
+const PayHeader = ({ step, onBackClick, onCloseClick }: Props) => {
+  let stepIcon;
+  switch (step) {
+    case 1:
+      stepIcon = Step1Icon;
+      break;
+    case 2:
+      stepIcon = Step2Icon;
+      break;
+    case 3:
+      stepIcon = Step3Icon;
+      break;
+  }
```

Comment:
```suggestion
const stepIcons = {
  1: Step1Icon,
  2: Step2Icon,
  3: Step3Icon,
} as const;

const stepIcon = stepIcons[step];
```

이런 식으로 객체 매핑을 사용하면 코드를 쪼오금 더 간결하게 작성할 수 있고 유지보수 하기도 편리합니닷

Reply:
헉 그러네ㅇ용!! 바로 수정완료해두었습니다! 좋은 코드 방향 잡아주셔서 감삼당